frontend/components/watch/card.js

```diff
@@ -1,26 +1,32 @@
-import { useState } from 'react'
-import axios from 'axios'
+import { useState, useContext } from 'react'
 import TruncateMarkup from 'react-truncate-markup'
+import { observer } from 'mobx-react-lite'
 
+import { watchListStoreContext } from '../../stores/watch-list'
 import { secondsToHumanTime, mostRecent } from '../../utils'
 
-const WatchCard = (props) => {
-  const { url, _id, interval, targets, checkedAt } = props
+const WatchCard = observer((props) => {
+  const { url, _id, interval, targets, checkedAt, active } = props
 
-  const [active, setActive] = useState(props.active)
+  // const [active, setActive] = useState(props.active)
+
+  const listWatchStore = useContext(watchListStoreContext)
```

Comment:
```suggestion
  const watchListStore = useContext(watchListStoreContext)
```
Còn chỗ nào thì đổi tên nốt dùm anh với.